### DISCUSSION TITLE
How do I launch apps with NFC?

### DISCUSSION THREAD

**Discussion by Hyun-Ha** [community/community#125860] [Programming Help]
### Body

If there is a launcher icon, it will be launched via NFC.

Remove app launcher icon 
The application installation has been completed without a launcher.

If you remove the launcher icon, NFC will not work.
How do I launch apps with NFC?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Hyun-Ha, thanks for being a part of the GitHub Community!

[This answer](https://stackoverflow.com/questions/30286702/how-to-launch-an-app-without-launcher-icon-by-scanning-an-nfc-tag) on Stacked Overflow is old but may still be relevant to help guide you.
